app/jobs/user_session_jobs/send_question_sms_job.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class UserSessionJobs::SendQuestionSmsJob < ApplicationJob
+  queue_as :question_sms
+
+  def perform(user_id, question_id, user_session_id)
+    user = User.find(user_id)
+
+    return unless user&.sms_notification || user.predefined_user_parameter&.sms_notification
```

Comment:
I believe this check isn't sufficient. We have duplicated information, and we should think about how to improve this. For now, we need to adjust this check to meet our criteria. Look at the attached screen; you can see that the user has defined sms_notification, but in the predefined user's options, SMS notifications are disabled. We need to prioritize the setting from PDD.

<img width="1890" alt="Screenshot 2024-05-20 at 08 26 44" src="https://github.com/Michigan-State-University/cias-api/assets/61981027/01d6b287-582a-4453-99a1-285efb945092">